data/ships.txt

```diff
@@ -1480,6 +1480,46 @@ ship "Gunboat"
 
 
 
+ship "Harvester"
+	sprite "ship/mining drone"
+	attributes
+		category "Drone"
+		"cost" 100000
+		"shields" 200
+		"hull" 500
+		"mass" 22
+		"drag" .6
+		"heat dissipation" .92
```

Comment:
0.92 is greater than both other human drones, each at 0.9. I'd think that this should either match or be beat out by that, dropping maybe to 0.85 if it doesn't match.